### PR TITLE
[new] scripts/update-schema-id

### PR DESCRIPTION
**Pull request by TakahikoKawasaki**
This script updates the schema IDs in the following YAML files:

- schemas/v{major}.{minor}/schema.yaml
- schemas/v{major}.{minor}/schema-base.yaml

and generates the following JSON files from the updated YAML files:

- schemas/v{major}.{minor}/schema.json
- schemas/v{major}.{minor}/schema-base.json

### CODE REVIEW

File: scripts/update-schema-id [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+      @quiet = true
+    end
+  end
+
+  private
+
+  def error_exit(message)
+    $stderr.puts message
+    exit false
+  end
```
Comment (notEthan) [inline scripts/update-schema-id:135]: built-in `abort` (method of Kernel) does the same as this `error_exit` method

Comment (TakahikoKawasaki) [reply scripts/update-schema-id]: @notEthan Thank you! I have replaced `error_exit` with `abort`.

**Comment by lornajane** [comment]
Thanks for getting us started with this change. We don't currently have any Ruby code in our repository, and it's not a skill present in our current maintainer cohort either. Could we discuss what this script does, and whether we can use a tech stack that is already in use?

**Comment by TakahikoKawasaki** [comment]
Okay. A shell script version is here --> PR #3814

**Comment by handrews** [comment]
Replaced by #3814

**Pull request closed without merging**